### DISCUSSION TITLE
How do I get the Mars helicopter badge?

### DISCUSSION THREAD

**Discussion by GH05T-HUNTER5** [community/community#65955] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![nasa-ingenuity-helicopter-spot-1_1618909948852_1618909976052](https://github.com/community/community/assets/108191615/936f991a-f94d-4a09-ad23-545b5fdd52ba)

**Comment by advayc** [comment, marked as answer]
Developers who contributed to the specific versions of projects and libraries used by NASA’s Jet Propulsion Laboratory to put the Ingenuity helicopter into the Martian atmosphere will receive a new badge on their profile to commemorate their contributions to this historic achievement in planetary exploration. 

Currently however, you cannot get this badge anymore, as it was only given out in 2020.

See the list of repositories and versions which qualify a user for getting the badge [here.](https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#list-of-qualifying-repositories-for-mars-2020-helicopter-contributor-badge)

**Reply by RenanSX** [reply]
👀

**Comment by radar-blip** [comment]
You can't get that badge anymore. It was only given out in 2020.

See here: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)
